From 65fda228e5e69c84438eb47de7c40d75a7bd0f4b Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Tue, 27 May 2025 19:32:44 +0300
Subject: [PATCH] [SW-229758] gaudi2: bump svn to 10

Supporting Gaudi2E requires bumping the SVN to 10.
(It won't be possible to downgrade G2E to G2D).

Change-Id: I25bea1fde91d63fd6091544efd10ee0e5db053cb
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/gaudi2/gaudi2_security.h b/gaudi2/gaudi2_security.h
index 8da26e5..b3da78e 100644
--- a/gaudi2/gaudi2_security.h
+++ b/gaudi2/gaudi2_security.h
@@ -10,7 +10,7 @@
 
 #include <hl_sec_if.h>
 
-#define HL_SEC_FW_VERSION			9
+#define HL_SEC_FW_VERSION			10
 #define HL_SEC_KEY_INDEX			0x0
 
 #define HL_SDRAM_BASE				0x11000000
